build.rs

```diff
@@ -690,30 +690,34 @@ fn find_interpreter_and_get_config() -> Result<(InterpreterConfig, BuildFlags)>
 /// Extract compilation vars from the specified interpreter.
 fn get_config_from_interpreter(interpreter: &Path) -> Result<InterpreterConfig> {
     let script = r#"
+import os.path
 import platform
 import struct
 import sys
 import sysconfig
-import os.path
 
 PYPY = platform.python_implementation() == "PyPy"
 
 # Anaconda based python distributions have a static python executable, but include
 # the shared library. Use the shared library for embedding to avoid rust trying to
 # LTO the static library (and failing with newer gcc's, because it is old).
-ANACONDA = os.path.exists(os.path.join(sys.base_prefix, 'conda-meta'))
+ANACONDA = os.path.exists(os.path.join(sys.base_prefix, "conda-meta"))
 
-libdir = sysconfig.get_config_var('LIBDIR')
+libdir = sysconfig.get_config_var("LIBDIR")
+ld_version = sysconfig.get_config_var("LDVERSION")
 
 print("version_major", sys.version_info[0])
 print("version_minor", sys.version_info[1])
 print("implementation", platform.python_implementation())
 if libdir is not None:
     print("libdir", libdir)
-print("ld_version", sysconfig.get_config_var('LDVERSION') or sysconfig.get_config_var('py_version_short'))
+print(
+    "ld_version",
+    ld_version or sysconfig.get_config_var("py_version_short"),
+)
```

Comment:
I formatted this by black with its default line width (=88).

Reply:
Might be able to reduce line width for this and the lines below by caching `sysconfig.get_config_var`, e.g.:
```suggestion
var = sysconfig.get_config_var
print("ld_version", ld_version or var("py_version_short"))
```

Reply:
Thanks, I choose `from sysconfig import get_config_var`.